PhysicsTools/PatAlgos/python/producersLayer1/muonProducer_cfi.py

```diff
@@ -107,7 +108,8 @@
     # Depends on MiniIsolation, so only works in miniaod
     # Don't forget to set flags properly in miniAOD_tools.py                      
     computeMuonMVA = cms.bool(False),
-    mvaTrainingFile = cms.FileInPath("RecoMuon/MuonIdentification/data/mu_BDTG_Run2017.weights.xml"),
+    mvaTrainingFile      = cms.FileInPath("RecoMuon/MuonIdentification/data/mu_2017_BDTG.weights.xml"),
```

Comment:
the new file names are nice, but I do not recognize them from recent PRs in https://github.com/cms-data/RecoMuon-MuonIdentification
Is it "coming soon"?

Reply:
Yes, they will be there in https://github.com/cms-data/RecoMuon-MuonIdentification/pull/6 when @bmahakud merges https://github.com/bmahakud/RecoMuon-MuonIdentification/pull/1. I didn't want to create a new PR and add more noise